package.json

```diff
@@ -85,7 +85,7 @@
     "test:node-test": "borp -p \"test/node-test/**/*.js\"",
     "test:tdd": "borp --expose-gc -p \"test/*.js\"",
     "test:tdd:node-test": "borp -p \"test/node-test/**/*.js\" -w",
-    "test:typescript": "tsd && tsc --skipLibCheck test/imports/undici-import.ts",
+    "test:typescript": "tsd && tsc test/imports/undici-import.ts --typeRoots ./types && tsc ./types/*.d.ts --noEmit --typeRoots ./types",
```

Comment:
eslint is throwing typings errors. to avoid it we simulate an skipLibCheck by pointing to the types folder. this is not nice but eslint sucks. So this is a low level solution.